src/aiofiles/tempfile/__init__.py

```diff
@@ -229,7 +242,7 @@ async def _temporary_directory(
 
 
 class AiofilesContextManagerTempDir(AiofilesContextManager):
-    """With returns the directory location, not the object (matching sync lib)"""
+    """Returns the directory location, not the object (matching sync lib)"""

```

Comment:
```suggestion
    """Return the directory location, not the object (matching sync lib)"""

```

Reply:
Returns or Return? Favouring to which version?